templates/admin/actions.tmpl

```diff
@@ -1,5 +1,7 @@
-{{template "admin/layout_head" (dict "ctxData" . "pageClass" "admin runners")}}
+{{template "admin/layout_head" (dict "ctxData" . "pageClass" "admin actions")}}
 	<div class="admin-setting-content">
+	{{if eq .PageType "runners"}}
```

Comment:
Why we need this check

Reply:
This is because in the future there might be more actions related settings, so prepared for this.. I will remove this for now.

Reply:
Then why do not make them have separate templates? I do not see a reason that these sub-templates should be mixed together and called by `if`